src/frontend/src/config.js

```diff
@@ -4,3 +4,26 @@ export const clearLocalStorage = () => window.localStorage.clear();
 export const retrieveConfig = (key) => JSON.parse(window.localStorage.getItem(key));
 export const storeConfig = (key, json) =>
   window.localStorage.setItem(key, JSON.stringify(json, null, "  "));
+
+export const networks = [
+  {
+    name: "Peak",
+    url: `${process.env.NETWORK_URL_ROOT}/peak`,
```

Comment:
We don't have to use the name `NETWORK_URL_ROOT`- let me know if you prefer something else

Reply:
This naming makes sense to me 👍

Reply:
Actually, what do you think about separating the year path from the NETWORK_URL_ROOT? If we have a separate TRANSIT_YEAR variable constant on the frontend it might be easier to reference/update after re-generating analyses in the future

Reply:
>Actually, what do you think about separating the year path from the NETWORK_URL_ROOT? If we have a separate TRANSIT_YEAR variable constant on the frontend it might be easier to reference/update after re-generating analyses in the future

Good question. Do we expect to support year selection in the app? If we do, then we should leave out the year part from the `NETWORK_URL_ROOT` env var. Otherwise, we might not have a use case for separating out the year. Assuming there is no year selection feature, after we have data from 2025, we will just update the `NETWORK_URL_ROOT` env var on S3 with the new year in the URL segment and rebuild the app. What do you think? Does it make sense?

Reply:
> Do we expect to support year selection in the app?

Nope! It's assumed we'll always use the latest transit analysis files and those will likely not update *that* often (annually, if that). Updating the env var on S3 makes sense 👍